video_frame_metadata_registry.src.html

```diff
@@ -61,6 +61,18 @@
     <td>segments</td>
     <td>[Human face segmentation](https://w3c.github.io/mediacapture-extensions/#human-face-segmentation)</td>
   </tr>
+  <tr>
+    <td>captureTime</td>
+    <td>[Capture time](https://wicg.github.io/video-rvfc/#dom-videoframecallbackmetadata-capturetime)</td>
```

Comment:
The RVFC text is too RTP centric. For a WebCodecs application, captureTime can be serialized on the wire and set by a WebCodecs application for frames received from a remote peer. Also, NTP timestamp format does not imply a global clock so change “synchronization” to “alignment”.

Reply:
The PR is now updated to reference mediacapture-extensions, where these concepts are now defined.